### PR TITLE
Documentation Improvements

### PR DESCRIPTION
**Pull request by freddykrunn**

### CODE REVIEW

File: generators/widget/templates/widgetSettings.ts [hunk 1]
```diff
@@ -13,15 +13,37 @@ import i18n from "./i18n/<%= widget.name %>.default";
 import * as ng from "@angular/core";
 
 /**
- * <%= widget.class %> Widget Settings
+ * @whatItDoes
  * 
- * Please provide a meaningful description of this widget and how to use it
+ * Please provide a meaningful description of this Widget Settings and what it is needed for, adn describe all the properties that are configurable in the correspondent Widget
```
Comment (jpsfs) [inline generators/widget/templates/widgetSettings.ts]: Small typo on "adn"

**Review by jpsfs — CHANGES_REQUESTED** [review]

File: generators/component/templates/component.ts [hunk 2]
```diff
@@ -22,51 +22,64 @@ import * as ng from "@angular/core";
...
  * Try to answer these questions: 
- * What is it?
- * What does it do?
- * Where to use it?
- * How to use it?
- * How it behaves with different sizes?
- * Does it retrieve data from any external source (server, local database, text file, etc...)?
+ * * What is it?
+ * * What it does?
+ * * How it behaves with different sizes?
```
Comment (francisco-maciel) [inline generators/component/templates/component.ts]: Change to _"How does it behave with different sizes?"_

File: generators/widget/templates/widget.ts [hunk 3]
```diff
@@ -21,35 +21,61 @@ let outputs: Map<string, Widget.WidgetOutput> = new Map<string, Widget.WidgetOut
...
+ * 
  * Please provide a meaningful description of this widget.
- * Try to answer these questions: 
+ * Try to answer these questions here: 
  * * What is it?
- * * What does it do?
- * * Where to use it?
- * * How to use it?
+ * * What it does?
  * * How it behave with different sizes?
```
Comment (francisco-maciel) [inline generators/widget/templates/widget.ts]: Change to _"How does it behave with different sizes?"_

File: generators/dataSource/templates/dataSource.ts [hunk 4]
```diff
@@ -9,10 +9,31 @@ import i18n from "./i18n/<%= dataSource.name %>.default";
...
 /**
- * <%= dataSource.class %> DataSource
+ * @whatItDoes
  * 
- * Please provide a meaningful description of this data data source
+ * Please provide a meaningful description of this DataSource.
+ * Try to answer these questions: 
+ * * What is it?
+ * * What it does?
+ * * How it behave with different sizes?
```
Comment (francisco-maciel) [inline generators/dataSource/templates/dataSource.ts]: I don't think a data source has _"sizes"_, i think you should remove this line.

**Review by francisco-maciel — CHANGES_REQUESTED** [review]

**Review by jpsfs — APPROVED** [review]

**Review by francisco-maciel — APPROVED** [review]

**Pull request merged**